### DISCUSSION TITLE
custom redirect

### DISCUSSION THREAD

**Discussion by babikriuk** [filamentphp/filament#9553] [Help] [answered]
I have a question, in documentation when we want to redirect after store data we use 
```
protected function getRedirectUrl(): string
{
    return $this->getResource()::getUrl('index');
}
```
for assumption we use this in post resource 
can we change that to other route for example from post.index to category.index..

**Comment by sinnbeck** [comment]
Just call the method on that resource
```
protected function getRedirectUrl(): string
{
    return CategoryResource::getUrl('index');
}

**Reply by babikriuk** [reply]
i get some error.. undefined function.. 
i already use App\Filament\Resources\CategoryResource;

**Reply by sinnbeck** [reply]
What error exactly ? Show your exact code please

**Reply by babikriuk** [reply]
`<?php

namespace App\Filament\Resources\OrderResource\Pages;

use App\Filament\Resources\CategoryResource;
use App\Filament\Resources\OrderResource;
use Filament\Actions;
use Filament\Resources\Pages\CreateRecord;
use Illuminate\Http\RedirectResponse;

class CreateOrder extends CreateRecord
{
    protected static string $resource = OrderResource::class;
    
    protected function getRedirectUrl(): string
    {
       
        return CategoryResource()::getUrl('index');
    }
}
`

**Reply by sinnbeck** [reply, marked as answer]
No ()

```
return CategoryResource::getUrl('index');

**Reply by babikriuk** [reply]
oh my.. thanks a lot...
